.changeset/lazy-oranges-sip.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Let's delete this one, since it's just a change for the tests. No need for a changelog entry (not user facing).